src/@carbon-info/views/Ecosystem/components/AppsCarousel.tsx

```diff
@@ -34,13 +34,15 @@ const AppsCarousel: React.FC<Props> = (props: Props) => {
         setView(currSlideMod);
       }}
       minimumTouchDrag={150}
+      transitionDuration={1000}
+      customTransition={"transform 1500ms cubic-bezier(.1,1.03,1,1) 0ms"}
```

Comment:
After testing, think the customTransition can be a bit, think duration can be reduced from `1500ms` => `1000ms`